src/ol-ext/interaction/modifyrectangle.js

```diff
@@ -191,11 +191,81 @@ ngeo.interaction.ModifyRectangle.prototype.willModifyFeatures_ = function(evt) {
     this.modified_ = true;
     this.dispatchEvent(new ol.interaction.Modify.Event(
       ol.interaction.ModifyEventType.MODIFYSTART, this.features_, evt));
+    this.params_ = this.initializeParams_();
```

Comment:
The idea was to delay the vector calculation until the resizing actually starts so we don't calculate the vectors for nothing each time the user clicks on a feature.